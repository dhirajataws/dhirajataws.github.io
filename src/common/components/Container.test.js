import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { Container } from './Container';

test('container components ', () => {
    const tree = renderer.create(<Container />).toJSON()
    expect(tree).toMatchSnapshot()
    expect(tree).toHaveStyleRule('display', 'flex')
    expect(tree).toHaveStyleRule('flex-wrap', 'wrap')
    expect(tree).toHaveStyleRule('justify-content', 'space-between')
})