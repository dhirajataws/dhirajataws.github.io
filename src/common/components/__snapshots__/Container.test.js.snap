// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`container components  1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

<div
  className="c0"
/>
`;
